bsdlib/doc/gnss_extension.rst

```diff
@@ -143,11 +143,57 @@ Satellites with elevation angle less than the threshold are automatically exclud
 Use case
    | Socket option:  :c:type:`NRF_SO_GNSS_USE_CASE`
    | Datatype:       :c:type:`uint8_t`
-   | Default         1
-   | Allowed values: 0, 1
+   | Default         :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START | NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | Allowed values:
+
+   | :c:type:`NRF_GNSS_USE_CASE_SINGLE_COLD_START`
+   | :c:type:`NRF_GNSS_USE_CASE_MULTIPLE_HOT_START`
+   | :c:type:`NRF_GNSS_USE_CASE_NORMAL_ACCURACY`
+   | :c:type:`NRF_GNSS_USE_CASE_LOW_ACCURACY`
+
+This bitmask is used to specify the combination of the target GNSS performance (single cold or multiple hot start)
+and accuracy (normal or low).
+
+..Note::
+  Normal and Low accuracy is available in MFW 1.2.2 and later
+
+Low Accuracy:
+
+Enabling low accuracy mode will result in the GNSS receiver having a looser acceptance criterion
+for a fix (i.e. error in calculation vs actual position can be larger than Normal Accuracy mode).
+In addition, in the Low Accuracy mode the GNSS receiver may use only 3 satellites to determine a
+fix (in Normal Accuracy mode, it needs 4 or more satellitesi).
+
+For a possible 3 satellite fix to take place, one of the following criteria must be in place first
+and will be valid for a time window of 24 hour:
+
+ | 1) At least 1 fix must have been taken place using 5 or more satellites. You initiate the GPS to start Low accuracy mode, but the radio will not be able to utilize 3 satellite fixing before the criteria above has taken place. In the following time window, any fix that is done using 5 or more satellites will reset the time window and extend the 24-hour period.
+ |
+ | 2) Injecting Altitude (in meter), Altitude uncertainty (0) , and Confidence (100 percent) to the GPS radio using agps data location struct as described below. To be able to stay in Low Accuracy mode this step has to be repeated at least every 24 hour.
```

Comment:
GPS radio -> GNSS engine (if that's the correct term).
` to stay in Low Accuracy mode this step has to be repeated at least every 24 hour`, unless there's a 5-or-more-satellite fix within that 24-hour time window?